docs/index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   canal:
   - apiVersion: v1
     appVersion: v3.13.3
-    created: "2020-06-22T19:23:35.619402732Z"
+    created: "2020-06-26T11:06:00.318061-07:00"
```

Comment:
@StrongMonkey it seems like all of the timestamps in`docs/index.yaml` are being updated on running `make CHART=rancher-monitoring-v2 charts` due to https://github.com/rancher/dev-charts/blob/f091a152fbe8ece1a05a4cb8e800aa217d48f825/scripts/generate-charts#L22

Is this expected?

Reply:
Yes, I think this is a known issue. Don't know how to fix it yet.